src/core/print_counts_probs.jl

```diff
@@ -1,7 +1,6 @@
-# The contents of this file is directly copied from DimensionalData.jl v0.25.2,
-# but modified slightly so that we can reduce the amount of unnecessary printing
-# for `Counts` and `Probabilities`.
-import DimensionalData: NoName
+# # The contents of this file is directly copied from DimensionalData.jl v0.25.2,
+# # but modified slightly so that we can reduce the amount of unnecessary printing
+# # for `Counts` and `Probabilities`.
```

Comment:
so this file can be deleted now, right? it is full of commented out code.

Reply:
See my comment below.